submarine-sdk/pysubmarine/submarine/job/README.md

```diff
@@ -0,0 +1,31 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
```

Comment:
I'd suggest that we add another index README for pysubmarine. It can have a link to this "how to generate REST SDK from swagger". The main content is to illustrate how we use submarine APIs including high-level ML API and low-level REST API. The outline can be like this:

```
#Title: Submarine SDK introduction
Submarine SDK is aiming to ease the ML engineer's life by providing a set of libraries.

It includes a high-level out-of-box ML library like deepFM .etc and low-level library to interact with submarine like creating experiment .etc.

## Easy-to-use model trainers
Link to documents like deepFM and FM

## Low-level APIs
The low-level APIs are exposing the abilities submarine have. For instance, experiment CRUD, environment CRUD, logging.
Link document to the ipynb
```

Reply:
done